### PR TITLE
Update Puppet Providers service

### PR DESCRIPTION
**Pull request by Soumi7**
Updates References - Wechat4u section in Puppet Providers

References issue #890

Screenshots :

image

image

image

image

### CODE REVIEW

**Review by shraddhavp — APPROVED** [review]
LGTM

File: docusaurus/docs/puppet-providers/service.md [hunk 1]
```diff
@@ -17,18 +17,18 @@ sidebar_label: Service
 
 Wechaty Puppet Service is a Wechaty Puppet for providing/consuming gRPC Puppet API. To use a Wechaty Puppet Service, you need a TOKEN for that service, and pass it to Wechaty, then you will be able to use that Puppet Service.
 
-If you want to learn more about the concepts behind TOKEN, please read our blog post: [Introducing Wechaty Puppet Service](https://wechaty.js.org/2021/01/14/wechaty-puppet-service/)
+For example, you can cloudify the Wechaty Puppet Provider wechaty-puppet-padlocal to a Wechaty Puppet Service by running our Wechaty Puppet Service Token Gateway.
 
-Learn all service provided by the Wechaty Community at [here](puppet-services/overview.mdx)
+Wechaty Puppet Service Token Gateway is a gateway for converting the Wechaty Puppet Provider to a Wechaty Puppet Service. Read more about tokens and usage [here](https://wechaty.js.org/2021/01/14/wechaty-puppet-service/).
+
+Learn about all puppet services provided by the Wechaty Community [here](puppet-services/overview.mdx).
```
Comment (proudofsimin) [inline docusaurus/docs/puppet-providers/service.md]:
```suggestion
Learn more about all puppet services provided by the Wechaty Community in [puppet service overview](puppet-services/overview.mdx).
```

File: docusaurus/docs/puppet-providers/service.md [hunk 2]
```diff
@@ -17,18 +17,18 @@ sidebar_label: Service
 
 Wechaty Puppet Service is a Wechaty Puppet for providing/consuming gRPC Puppet API. To use a Wechaty Puppet Service, you need a TOKEN for that service, and pass it to Wechaty, then you will be able to use that Puppet Service.
 
-If you want to learn more about the concepts behind TOKEN, please read our blog post: [Introducing Wechaty Puppet Service](https://wechaty.js.org/2021/01/14/wechaty-puppet-service/)
+For example, you can cloudify the Wechaty Puppet Provider wechaty-puppet-padlocal to a Wechaty Puppet Service by running our Wechaty Puppet Service Token Gateway.
 
-Learn all service provided by the Wechaty Community at [here](puppet-services/overview.mdx)
+Wechaty Puppet Service Token Gateway is a gateway for converting the Wechaty Puppet Provider to a Wechaty Puppet Service. Read more about tokens and usage [here](https://wechaty.js.org/2021/01/14/wechaty-puppet-service/).
```
Comment (proudofsimin) [inline docusaurus/docs/puppet-providers/service.md]:
```suggestion
Wechaty Puppet Service Token Gateway is a gateway for converting the Wechaty Puppet Provider to a Wechaty Puppet Service. Read more about tokens and usage in [Introducing Wechaty Puppet Service (Providers)](https://wechaty.js.org/2021/01/14/wechaty-puppet-service/).
```

**Review by proudofsimin — APPROVED** [review]
add one sentence before the code snippet in the Usage: Run `wechaty-puppet-service`.

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Pull request merged**